changelog_unreleased/typescript/10961.md

```diff
@@ -0,0 +1,18 @@
+#### Fix order of `override` modifiers (#10961 by @sosukesuzuki)
+
+```ts
+// Input
+class Foo extends Bar {
+  abstract override foo: string;
+}
+
+// Prettier stable
+class Foo extends Bar {
+  override abstract foo: string;
```

Comment:
[CHANGELOG.md#fix-order-of-override-modifiers-10961-by-sosukesuzuki](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#fix-order-of-override-modifiers-10961-by-sosukesuzuki) has this flipped, and it makes no sense anymore.

Blame: https://github.com/prettier/prettier/commit/a34b4a711ed9928f8809bed3f4c8572dc3a40efb